.github/workflows/iso.yaml

```
@@ -44,7 +44,7 @@ jobs:
     name: arkes:${{ inputs.variant }} iso
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/eeems/arkes-builder:${{ inputs.builder || github.head_ref || github.ref_name }}
+      image: ghcr.io/eeems/arkes-builder:${{ inputs.builder || replace(replace(replace(replace(replace(replace(github.head_ref || github.ref_name, ' ', '-'),'/', '-'),'@', '-'),':', '-'),'^[.]+', ''),'^-]+', '')}}
```

actionlint (1.7.11)

[error] undefined function "replace". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)